### DISCUSSION TITLE
ERROR in Conflict: Multiple assets emit different content to the same filename native/pty.node

### DISCUSSION THREAD

**Discussion by lqd1434** [eclipse-theia/theia#13959] [Q&A]
**When i upgrade theia from 1.45.1 to 1.49.1 in our app, i meet this error**

full log:
![QQ_1721907568391](https://github.com/user-attachments/assets/0fc25aa3-abe2-4dd0-bdbd-bb477b38c0da)

**Comment by msujew** [comment]
Hey @lqd1434,

do you have a reproducible example at hand? There is likely some issue in your custom webpack config. But just from the log, it's impossible to tell what exactly is wrong.

**Comment by lqd1434** [comment]
@msujew,  Thank you for your reply, I fallback all custom config, and the config just same as theia, also get same error, but i notice the build webpack version is 5.76.2, but i found theia is 5.90.0, maybe that's the problem, i will try in tomorrow.

**Comment by lqd1434** [comment]
@msujew I fallback all coustom config and now the config just same as theia, and i upgrade webpack to 5.90.0, unfortunately, i got same error, there is no other way. i have to change the webpack rules
`
 {
    test: /\\.node$/,
    loader: 'node-loader',
    options: {
        name: 'native/[name].[hash].[ext]'
     }
 },`
and **add a hash to the output name**,  then , this error was solved，I don't know if this is feasible，do you have any idea ?
